Project1/application.py

```diff
@@ -0,0 +1,62 @@
+import os
+
+from flask import Flask, session,url_for,redirect
+from flask import Flask,render_template,request,flash
+from flask_session import Session
+from  datetime import datetime
+# from Database import *
+
+app = Flask(__name__)
+app.secret_key = "secret"
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+
+@app.route('/')
+def indexed():
+    if 'username' in session:
```

Comment:
Why have you implemented sessions when the first task is to just to redirect to /register route. Redirect / to /register